README.md

```diff
@@ -204,10 +204,13 @@ The torch wheel version `2.6.0.dev20240925+cpu` can be found at https://download
 
 | Version | GPU Wheel |
 | --- | ----------- |
-| 2.5 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
+| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
-| 2.4 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
```

Comment:
Did the 2.4 URL never work in the first place so now you remove it? I visited https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl and got 404.

Reply:
when I check, I found no trigger for py38 for 2.4.0, so update it to exitsing trigger py39 for 2.4.0, and we don't have this wheel too, its more like a cleanup